packages/perseus/package.json

```diff
@@ -38,6 +38,7 @@
         "@khanacademy/wonder-blocks-dropdown": "^3.0.8",
         "@khanacademy/wonder-blocks-form": "^4.0.8",
         "@khanacademy/wonder-blocks-i18n": "^2.0.2",
+        "@khanacademy/wonder-blocks-icon": "^2.0.13",
```

Comment:
I don't see this being used in the rest of this PR.  Is it still needed?

Reply:
Thanks for pointing this out! I ended up using a custom icon anyway.